staging/src/k8s.io/kubectl/pkg/explain/v2/explain_test.go

```diff
@@ -0,0 +1,184 @@
+var apiGroupsGVR schema.GroupVersionResource = schema.GroupVersionResource{
+	Group:    "discovery.k8s.io",
+	Version:  "v1",
+	Resource: "apigroups",
+}
+var apiGroupsDocument map[string]interface{} = func() map[string]interface{} {
+	var doc map[string]interface{}
+
+	err := json.Unmarshal([]byte(apiDiscoveryJSON), &doc)
+	if err != nil {
+		panic(err)
+	}
+
+	return doc
+}()
+
+type FakeOpenAPIV3Client struct {
+	// Path:
+	//		ContentType:
+	//			OpenAPIV3 Schema bytes
+	Values      map[string]map[string][]byte
+	FetchCounts map[string]map[string]int
+	lock        sync.Mutex
+}
+
+type FakeGroupVersion struct {
+	Data        map[string][]byte
+	FetchCounts map[string]int
+	Lock        *sync.Mutex
+}
+
+func (f *FakeGroupVersion) Schema(contentType string) ([]byte, error) {
+	f.Lock.Lock()
+	defer f.Lock.Unlock()
+
+	if count, ok := f.FetchCounts[contentType]; ok {
+		f.FetchCounts[contentType] = count + 1
+	} else {
+		f.FetchCounts[contentType] = 1
+	}
```

Comment:
Nit, but Golang will return the zero value if it's not in the  map, so you can always just do `f.FetchCounts[contentType] += 1`, or am I reading this too fast?